pro.md

```diff
@@ -72,3 +72,22 @@ button.setOnClickListener(new View.OnClickListener() {
 </figure>
 
 The PIN value will never leave the device except in the USSD session. If the action has no PIN, but you want to skip the confirm screen you can pass any string value for the pin: `.extra("pin", "1234")`.
```

Comment:
We changed this in previous update so confirm doesn't show for Pro users right? Mind updating/removing this.